app/controllers/admins_controller.rb

```diff
@@ -99,6 +83,7 @@ def unban_user
   # POST /admins/approve/:user_uid
   def approve
     @user.remove_role :pending
+    @user.add_role :user
```

Comment:
Why is this needed? Isn't the the `user` role already added to the user when they are created?